src/modules/musig/main_impl.h

```diff
@@ -45,64 +45,58 @@ static void secp256k1_musig_sha256_init_tagged(secp256k1_sha256 *sha) {
     sha->bytes = 64;
 }
 
-/* Compute r = SHA256(ell, idx). The four bytes of idx are serialized least significant byte first. */
-static void secp256k1_musig_coefficient(secp256k1_scalar *r, const unsigned char *ell, uint32_t idx) {
+/* Compute MuSig coefficient which is constant 1 for the second pubkey and
+ * SHA256(ell, x) otherwise. second_pk_x can be NULL in case there is no
+ * second_pk. Assumes both field elements x and second_pk_x are normalized. */
+static void secp256k1_musig_coefficient_internal(secp256k1_scalar *r, const unsigned char *ell, secp256k1_fe *x, const secp256k1_fe *second_pk_x) {
```

Comment:
1. Contrary to what the comment says, I don't think `second_pk_x` can be null, because `secp256k1_fe_cmp_var` (https://github.com/ElementsProject/secp256k1-zkp/blob/f3708a1ecb445b1b05a0f8fcd1da6a88f83d89c4/src/field_5x52_impl.h#L266) does not expect it to be null.
2. `x` does not seems modified, shouldn't it be const?

Reply:
Thanks!

1. Correct, this comment wasn't updated when I changed the interface. Fixed.
2. Fixed